lib/miasma/contrib/aws/orchestration.rb

```diff
@@ -143,9 +143,18 @@ def load_stack_data(stack=nil)
         # @return [Models::Orchestration::Stack]
         def stack_save(stack)
           params = Smash.new('StackName' => stack.name)
+          if(stack.dirty?(:parameters))
+            initial_parameters = stack.data[:parameters]
```

Comment:
Currently testing this out. It looks like on new stack stack launch with sfn `initial_parameters` on L127 is being set to nil. 

Partial trace is:

```
/usr/local/opt/rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/bundler/gems/miasma-aws-67c47cfaa3d4/lib/miasma/contrib/aws/orchestration.rb:153:in `block in stack_save'
/usr/local/opt/rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/bundler/gems/miasma-aws-67c47cfaa3d4/lib/miasma/contrib/aws/orchestration.rb:151:in `each'
/usr/local/opt/rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/bundler/gems/miasma-aws-67c47cfaa3d4/lib/miasma/contrib/aws/orchestration.rb:151:in `each_with_index'
/usr/local/opt/rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/bundler/gems/miasma-aws-67c47cfaa3d4/lib/miasma/contrib/aws/orchestration.rb:151:in `stack_save'
/usr/local/opt/rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/gems/miasma-0.3.2/lib/miasma/models/orchestration/stack.rb:116:in `perform_save'
/usr/local/opt/rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/gems/miasma-0.3.2/lib/miasma/models/orchestration/stack.rb:109:in `save'
```

Tweaking L147 to read `initial_parameters = stack.data[:parameters] || {}` allows 'sfn create' to succeed.